login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="login-container">
+<div class="login-inputs"> 
+    <img class="logo-wolox-login" src="images/LogoWolox-Original.png" alt= "Wolox Logotype">
+    <div class="form-container"> 
+            <form class="form-login" action="/index.html" id="form-login">
+                <label class="form-label" for="email">Email</label>
+                <input class="form-input" type="text" id="email" name="email"><br><br>
```

Comment:
Avoid to use <br> for split controls... Use margin
For this, try insert each label and input into a div